### PR TITLE
make it possible to change aspect ratios

### PR DESCRIPTION
**Pull request by kentis**

### CODE REVIEW

File: styles/cp-editor.css [hunk 1]
```diff
@@ -163,6 +162,11 @@
   content: "\e990";
 }
 
+.h5p-course-presentation .h5p-slidecontrols-button.h5p-slidecontrols-button-aspect-ratio:before {
```
Comment (boyum) [inline styles/cp-editor.css:165]: Should we add a TODO statement for changing this to a more general solution when we have sorted out font icons?

File: styles/toolbar.css [hunk 2]
```diff
@@ -1,8 +1,8 @@
 .h5p-dragnbar {
-  height: 7%;
+  height: 1.5em;
```
Comment (boyum) [inline styles/toolbar.css:2]: I assume this is done to prevent the buttons changing size when changing the size of the course presentation itself?

Comment (kentis) [reply styles/toolbar.css:2]: jepp

File: scripts/cp-editor.js [hunk 3]
```diff
@@ -325,6 +329,33 @@ H5PEditor.CoursePresentation.prototype.appendTo = function ($wrapper) {
...
+          break;
+        case "3-4":
+          that.cp.slides.forEach(slide => slide.aspectRatio = "16-9");
+          that.cp.defaultAspectRatio = "16-9"
+          break;
+        default:
+          that.cp.slides.forEach(slide => slide.aspectRatio = "16-9");
+          that.cp.defaultAspectRatio = "16-9"
+          break;
+      }
```
Comment (boyum) [inline scripts/cp-editor.js]:
```suggestion
    .click(function(){
    	const aspectRatios = ["16-9", "9-16", "4-3", "3-4"];
        const slide = that.cp.slides[that.cp.$current.index()];
        const currentAspectRatioIndex = aspectRatios.indexOf(that.cp.defaultAspectRatio);
        const nextAspectRatioIndex = (currentAspectRatioIndex + 1) % aspectRatios.length;
        const nextAspectRatio = aspectRatios[nextAspectRatioIndex];
        
        that.cp.slides.forEach(slide => slide.aspectRatio = nextAspectRatio);
        that.cp.defaultAspectRatio = "nextAspectRatio;
```

Is this a more cycle-y way to do this? A bit easier to add or subtract dimensions.

Comment (boyum) [reply scripts/cp-editor.js]: ⚠️ The code is not in any way tested ⚠️

File: scripts/cp-editor.js [hunk 4]
```diff
@@ -2168,7 +2201,7 @@ H5PEditor.CoursePresentation.findField = function (name, fields) {
 };
 
 /** @constant {Number} */
-H5PEditor.CoursePresentation.RATIO_SURFACE = 16 / 9;
+H5PEditor.CoursePresentation.RATIO_SURFACE = 9/16; //16 / 9;
```
Comment (boyum) [inline scripts/cp-editor.js]: ?

File: styles/cp-editor.css (hunk 1 above)
Comment (kentis) [reply styles/cp-editor.css:165]: No need. This icon is likely to move quite soon anyway

**Pull request merged**